common/pinot/pinotQueryValidator_test.go

```diff
@@ -42,10 +42,17 @@ func TestValidateQuery(t *testing.T) {
 			query:     "",
 			validated: "",
 		},
-		"Case2: simple query": {
+		"Case2-1: simple query": {
 			query:     "WorkflowID = 'wid'",
 			validated: "WorkflowID = 'wid'",
 		},
+		"Case2-2: simple query with partial match": {
+			query:     "WorkflowID like 'wid'",
+			validated: "TEXT_MATCH(WorkflowID, wid)",
```

Comment:
According to the doc: https://docs.pinot.apache.org/basics/indexing/text-search-support, we still need single quotes to surround the text we are searching.

Reply:
good callout! updated it